ansible/group_vars/all/websites.yml

```diff
@@ -239,6 +239,15 @@ websites:
     custom_config: true
     state: "present"
 
+  - name: "execut-api.{{ canonical_hostname}}"
+    user: "symposium"
+    alternative_names:
+      - "api.execut.nl"
+    extra_includes:
+      - "execut-referer-tracking"
```

Comment:
`extra_includes` only works for websites that don't have `custom_config` enabled. If we need this include (which I doubt), we need to include it in the nginx configuration file